compiler/record-minmax/src/RecordMinMax.cpp

```diff
@@ -38,6 +38,15 @@ using DataType = luci_interpreter::DataType;
 namespace
 {
 
+void readDataFromFile(const std::string &filename, char *data, size_t data_size)
+{
```

Comment:
it would be nice to have some validation check
- is data valid?
- is data_size smaller than the file size?